src/Forms/Prism.Forms/Navigation/INavigationResult.cs

```diff
@@ -2,17 +2,29 @@
 
 namespace Prism.Navigation
 {
+    /// <summary>
+    /// Provides navigation results indicating whether the navigation was successful or if there was an encountered <see cref="Exception"/>.
+    /// </summary>
     public interface INavigationResult
     {
+        /// <summary>
+        /// <see langword="true"/> if the navigation was successful, <see langword="false"/> otherwise.
```

Comment:
May be false with a null exception if CanNavigate returned false

Reply:
Not sure I'm following your correction here. If CanNavigate returns false, then that is not a successful navigation so the summary for `Success` still sounds correct. 

The summary for `INavigationResult` isn't correct however and I think it'd be best to update it to be a bit more generic, something like "Provides navigation results indicating if the navigation was successful and any exceptions caught".

What do you think?

Reply:
I went ahead and made those changes. In the descriptions of INavigationResult it now doesn't imply there is an exception when success is false.

Reply:
@dansiegel Are you okay with these changes, can we get this PR merged?